apps/wing-console/console/ui/src/use-localstorage.ts

```diff
@@ -0,0 +1,79 @@
+// Imported and modified from https://github.com/streamich/react-use/blob/master/src/useLocalStorage.ts
+import type { Dispatch, SetStateAction } from "react";
+import {
+  useCallback,
+  useState,
+  useRef,
+  useLayoutEffect,
+  useEffect,
+  useMemo,
+} from "react";
+
+import { trpc } from "./trpc.js";
+
+const noop = () => {};
+
+export const useLocalStorage = <T>(
```

Comment:
We have `react-use` installed. Is it possible to use https://github.com/streamich/react-use/blob/master/docs/useLocalStorage.md?